code/modules/mob/living/simple_animal/abnormality/_auxiliary_modes/joke/ego_armor.dm

```diff
@@ -43,3 +43,17 @@
 							TEMPERANCE_ATTRIBUTE = 80,
 							JUSTICE_ATTRIBUTE = 80
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/aleph/sukuna
+	name = "heian era robes"
+	desc = "Wearing these makes you want to dismember a white haired sorcerer."
+	icon_state = "uniqueoffice4"
+	icon = 'icons/obj/clothing/ego_gear/custom_fixer.dmi'
+	worn_icon =  'icons/mob/clothing/ego_gear/custom_fixer.dmi'
+	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 40, BLACK_DAMAGE = 80, PALE_DAMAGE = 80) // 280
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 120,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
```

Comment:
These numbers need to be lowered. The 100 PE cost means that these will show up in things like wishing well, so it should have standard stats for an E.G.O. armor. This stat spread is currently nearly as good as if not better than twilight.